ewm-service/src/main/java/ru/practicum/ewm_service/rating/model/Rate.java

```diff
@@ -0,0 +1,33 @@
+package ru.practicum.ewm_service.rating.model;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+import ru.practicum.ewm_service.events.model.Event;
+import ru.practicum.ewm_service.user.model.User;
+
+import javax.persistence.*;
+
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@Data
```

Comment:
В аннотации Data есть реализация методов equals & hashCode и это очень важные методы для сущности БД, поэтому не стоит отдавать их реализацию на откуп сторонней библиотеки ломбок, а лучше сделать их самостоятельно. В целом  достаточно вместо data ставить Getter & Setter

Подробнее здесь:
https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/

ключевое ниже

Your equals() method should return false if the primary key value of both entity objects is null. But Lombok’s equals() method returns true. Because of that, you can’t add two new entity objects to a Set

Reply:
public class EndpointHit 

Этот класс не исправлен, поправьте пож-та